packages/vuepress-theme-odyssey/styles/base/_fonts.scss

```diff
@@ -0,0 +1,61 @@
+// Copyright 2018-Present Okta, Inc.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@font-face {
+  font-family: 'Whyte';
+  font-style: normal;
+  font-weight: normal;
+  src: url('https://ok3static.oktacdn.com/font/ABCWhyte-Book.woff') format('woff');
+}
+
+@font-face {
+  font-family: 'Whyte';
+  font-style: italic;
+  font-weight: normal;
+  src: url('https://ok3static.oktacdn.com/font/ABCWhyte-BookItalic.woff') format('woff');
+}
+
+@font-face {
+  font-family: 'Whyte';
+  font-style: normal;
+  font-weight: 600;
+  src: url('https://ok3static.oktacdn.com/font/ABCWhyte-Medium.woff') format('woff');
+}
+
+@font-face {
+  font-family: 'Whyte';
+  font-style: normal;
+  font-weight: 600;
+  src: url('https://ok3static.oktacdn.com/font/ABCWhyte-Medium.woff') format('woff');
+}
+
+@font-face {
+  font-family: 'Whyte';
+  font-style: italic;
+  font-weight: 600;
+  src: url('https://ok3static.oktacdn.com/font/ABCWhyte-MediumItalic.woff') format('woff');
+}
+
+@font-face {
+  font-family: 'Whyte';
+  font-style: italic;
+  font-weight: 600;
+  src: url('https://ok3static.oktacdn.com/font/ABCWhyte-MediumItalic.woff') format('woff');
+}
+
+// TODO: Inconsolata was just selected by our brand team as
+// Okta's font of choice. Work to ensure it gets put in the
+// CDN like Whyte has.
+
+@import url('https://fonts.googleapis.com/css2?family=Inconsolata&display=swap');
```

Comment:
> Same question here re: our license and Google's.
> 
> _Originally posted by @edburyenegren-okta in https://github.com/okta/odyssey/pull/714#discussion_r509666318_

Reply:
Same comment as above - I'd assume we'd need to specify this as a dependency in our **Third Party Notices** file.

Reply:
I have temporarily removed this file while the question is sorted out.

Reply:
Wanted to confirm we have the ability to surface this w/o third party notices.